### DISCUSSION TITLE
things that I miss...

### DISCUSSION THREAD

**Discussion by doreks** [Eugeny/tabby#6000] [Q&A]
these are some first impressions after starting to work with the customer

1/ on the one hand, after enabling the pluging to save logs, I miss to be able to activate and deactivate it for each specific session and to be able to select the specific storage location.

2/ something that I consider fundamental, maybe I am too weak, would be to be able to enable for each active session the option to search. When you are working with a lot of logs and/or changes, being able to search in the already printed lines for a value or executed string helps a lot.

I hope my comments are helpful

thanks!

**Comment by Eugeny** [comment]
2: Do you mean normal text search? It's there, press the `search` hotkey to show the search box

**Reply by doreks** [reply]
yes indeed, some kind of context menu, as well as it allows the option to duplicate screen so maybe a shortcut icon to certain "important" commands, search for example. I know there is a shortcut but you know...
it's just an idea :-)
